scalafix-interfaces/src/main/java/scalafix/interfaces/EvaluationError.java

```diff
@@ -0,0 +1,16 @@
+package scalafix.interfaces;
+
+public enum EvaluationError {
```

Comment:
It's a bit verbose, but shouldn't these new enums follow the current pattern and be prefixed by `Scalafix`  for consistency?

Reply:
yes. ok